packages/design-system/src/components/Table/TableBody.jsx

```diff
@@ -15,7 +15,12 @@ export const TableBody = ({ children, _stackable, ...tableBodyProps }) => {
     });
   };
 
-  return <tbody {...tableBodyProps}>{_stackable ? renderChildren() : children}</tbody>;
+  /* eslint-disable jsx-a11y/no-redundant-roles */
```

Comment:
Could you add a comment for reenabling this eslint rule after this block of code? Generally if you use `eslint-disable` you want to also use `eslint-enable`